tasks/proposals/gigaeth-launch.ts

```diff
@@ -0,0 +1,366 @@
+    return;
+  }
+
+  const chainlinkConf = config.ChainlinkAggregator[network];
+  if (!chainlinkConf) {
+    console.log(chalk.red(`'${network}': chainlink configuration not found`));
+    exit(1);
+  }
+
+  console.log("---------> register assets");
+  let deployer;
+  try {
+    deployer =
+      config.ATokensAndRatesHelper ||
+      (await hre.deployments.get("ATokensAndRatesHelper")).address;
+  } catch (error) {
+    // If not found, use the PoolConfigurator
+    deployer = await poolAddressesProvider.getPoolConfigurator();
+  }
+  console.log("Deployer Address:", deployer);
+  let nonce = await hre.ethers.provider.getTransactionCount(deployer);
+
+  console.log("---------> register GETH");
+  let agEthToken = utils.getContractAddress({
+    from: deployer,
+    nonce: nonce,
+  });
+  let reserveAddress = await getReserveAddress(config, "GETH");
+  if (agEthToken) {
+    const underlying = new hre.ethers.Contract(
+      reserveAddress,
+      (await hre.deployments.getArtifact("AToken")).abi,
+      signer
+    );
+    txs.push(
+      hydrationTx.assetRegistry.register(
+        ...Object.values({
+          id: gETH,
+          name: "GIGAETH",
+          assetType: "Erc20",
+          existentialDeposit: 1,
+          symbol: "GETH",
+          decimals: 18,
+          location: location(agEthToken),
+          xcmRateLimit: null,
+          isSufficient: true,
+        })
+      )
+    );
+  } else {
+    return Error("GETH ATOKEN DOESNT EXIST");
+  }
+
+  console.log("---------> register aETH");
+  nonce = await hre.ethers.provider.getTransactionCount(deployer);
+  let aEthToken = utils.getContractAddress({
+    from: deployer,
+    nonce: nonce + 3,
+  });
+  reserveAddress = await getReserveAddress(config, "ETH");
+  if (aEthToken) {
+    const underlying = new hre.ethers.Contract(
+      reserveAddress,
+      (await hre.deployments.getArtifact("AToken")).abi,
+      signer
+    );
+    txs.push(
+      hydrationTx.assetRegistry.register(
+        ...Object.values({
+          id: aETH,
+          name: "aETH",
+          assetType: "Erc20",
+          existentialDeposit: 0,
+          symbol: "aETH",
+          decimals: 18,
+          location: location(aEthToken),
+          xcmRateLimit: null,
+          isSufficient: true,
+        })
+      )
+    );
+  } else {
+    return Error("ETH ATOKEN DOESNT EXIST");
+  }
+
+  console.log("---------> register 2-Pool-gETH");
+  txs.push(
+    hydrationTx.assetRegistry.register(
+      ...Object.values({
+        id: gETHs,
+        name: "2-Pool-GETH",
+        assetType: "StableSwap",
+        existentialDeposit: 1,
+        symbol: "2-Pool-GETH",
+        decimals: 18,
+        location: null,
+        xcmRateLimit: null,
+        isSufficient: true,
+      })
+    )
+  );
+
+  console.log("update rate strategies");
+  await hre.run("review-rate-strategies", {
+    deploy: true,
+    fix: true,
+    batch: true,
+  });
+
+  for await (const el of getBatch()) {
```

Comment:
do this only on end of the root evm call block, ideally i would just use dispatch as aave admin instead of generic dispatch as